### DISCUSSION TITLE
Migration from JYERS Ender 3 V2

### DISCUSSION THREAD

**Discussion by hpolack** [mriscoc/Ender3V2S1#1138] [General]
I finally got to rebuilding my abandoned Ender 3 V2 and I believe it is finally functional after having only printed a couple items successfully with it in the past 3 years.  I have had a number of problems with the V2  which I won't bore everyone with the details, but I might ask for advice after my current issue is resolved.   I must have migrated to JYERS a2.0.1 at some point, but I specifically remember working with Mriscoc build and getting regular updates.  Now I want to get back to MRISCOC firmware and I wondered if I need to to do the display screen SD Card as well as the firmware on the motherboard or one before the other?   I know I updated the firmware at some point on the display, downgraded it to Creality and upgraded it again, but I don't remmber if it was for JYERS or MRISCOC and if it really dosen't matter.  My current thought is to update the motherboard firmware to MRISCOC and see what happens on the display.

Thank you for your help in advance.

**Comment by wazi555** [comment]
Hey

As far as i am aware you will not have to do the firmware on the screen, only if you want the custom graphics.
I did the main board upgrade on my V2 to a SKR e3v3 and have never looked back. I am now playing with marlin to try enable more of the possible features on the motherboard.

I would give it a shot, update the firmware and see what happens.

**Reply by hpolack** [reply]
Yep, you were right.   I was having all kinds of headaches setting the Z offset and movement in JYERS.  After messing around for a while and recalibrating everything and still having issues, I just grabbed the BIN for my printer and an SD 8mb and did it.  Man is it much better.  There is no delay when adjusting the XYZ live and much better controls everywhere.   I won't look back myself.  The only thing I think it needs is a built in tool for eStep like the Community 6.1 firmware I have on my CR6 SE.    I do acknowledge the fixed 100m extruder menu item, but the calculator is even better.  It guides you through and makes all the calculations.
